src/components/NewWorkflow/AddPhase/AddSubTask.js

```diff
@@ -1,21 +1,22 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 import {connect} from 'react-redux';
 
 
 
 class AddTaskCard extends Component {
 
+    // holds an array of what is to be displayed
     displayTask = [];
 
     state = {
         showdata: this.displayTask,
         post: ""
     }
 
-
+    //adds a new sub task card to end of list
     appendData=()=>{
         this.displayTask.push(<div key={Math.random()} className="subtask-display">
```

Comment:
can we .push into arrays with react? We may have to use whatever the ellipsis adding to arrays is called:
" this.setState({...this.state.blahblahblah, thing: 'stuff and things'}) "

Reply:
yeha there can be arrays and objects and variables and such that exist without being in state, it works like a function in the since we call this.function we also can call this.array
when doing this we dont change state so we can directly mutate them, usualy i only use this if i want some default values not to remove when state refreshes